.github/workflows/build-release-binaries.yml

```diff
@@ -0,0 +1,98 @@
+            - "*.*.*"
+            # branches: [testaction]
+
+jobs:
+    build:
+        runs-on: macOS-latest
+
+        strategy:
+            matrix:
+                node-version: [16.13.x]
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - run: echo "PackageVersion=$(cat package.json | sed 's/,/\n/g' | grep '\"version\":' | sed 's/:/\n/g' | sed '1d' | sed 's/ //g' | sed 's/\"//g' | sed 's/}//g')" >> $GITHUB_ENV
+            - name: Update package.json
+              run: |
+                  sed -i -e "s/${{ env.PackageVersion }}/${{ github.ref_name }}/g" package.json
+            - run: yarn
+            - name: Build package
+              run: npm run prepackage
+            - name: Build dmg
+              run: ./node_modules/.bin/electron-builder -p never --mac dmg
+            - name: Build deb
+              run: ./node_modules/.bin/electron-builder -p never --linux deb
+            - name: Build exe
+              run: ./node_modules/.bin/electron-builder -p never --windows
+            - name: Calc hash
+              id: calc_hash
+              run: |
+                  echo "::set-output name=dmghash::${{ hashFiles('build/binaries/**.dmg') }}"
+                  echo "::set-output name=debhash::${{ hashFiles('build/binaries/**.deb') }}"
+                  echo "::set-output name=exehash::${{ hashFiles('build/binaries/**.exe') }}"
+            - name: Create Body
+              id: create_body
+              uses: xiangxn/generate_release_notes@v1
+              with:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  categories: |
+                      [{"title":"New features","labels":["feature","enhancement"]},{"title":"Bug fixes and improvements","labels":["bug","bugfix"]}]
+                  template: |
+                      Release Notes
+                      _________________________________
+
+                      $CHANGES
+
+                      Binaries for download
+                      --------
+                      | Platform | SHA256 Checksum |
+                      |---|---|
+                      |[Microsoft Windows](https://github.com/bitshares/bitshares-ui/releases/download/${{ github.ref_name }}/BitShares_${{ github.ref_name }}.exe)|`${{steps.calc_hash.outputs.exehash}}`|
+                      |[Mac](https://github.com/bitshares/bitshares-ui/releases/download/${{ github.ref_name }}/BitShares_${{ github.ref_name }}.dmg)|`${{steps.calc_hash.outputs.dmghash}}`|
+                      |[Linux](https://github.com/bitshares/bitshares-ui/releases/download/${{ github.ref_name }}/BitShares_${{ github.ref_name }}.deb)|`${{steps.calc_hash.outputs.debhash}}`|
+            - name: Create Release
+              id: create_release
+              uses: actions/create-release@v1.1.4
```

Comment:
Note that https://github.com/actions/create-release is unmaintained. We can still use it as long as it works though.

By the way, it's better to use a short version (E.G. `@v1`) because usually it will be updated to the latest minor version when there is a new one.

Reply:
It mentions maintained versions in there as well 

Maintained Actions:

elgohr/Github-Release-Action
marvinpinto/action-automatic-releases
softprops/action-gh-release
ncipollo/release-action

Can we re-use an existing one, or do we have custom needs? Less effort in the long run.

Reply:
If the short version is always the highest version, I can use such as @v1

Reply:
@sschiessl-bcp IMHO if a tool works fine, we can still use it even if it's not maintained, because sometimes it is "good enough" so there is little need to maintain. The ones currently being maintained can be unmaintained some day in the future too. We can look for alternatives if it no longer meets out needs. Nothing is perfect. Progress is more important.